src/main/java/runge/weather/CurrentWeatherFrame.java

```diff
@@ -0,0 +1,55 @@
+package runge.weather;
+
+import retrofit2.Retrofit;
+import retrofit2.adapter.rxjava3.RxJava3CallAdapterFactory;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class CurrentWeatherFrame extends JFrame
+{
+    public CurrentWeatherFrame()
+    {
+        Retrofit retrofit = new Retrofit.Builder()
+                .baseUrl("https://api.openweathermap.org/")
+                .addConverterFactory(GsonConverterFactory.create())
+                .addCallAdapterFactory(RxJava3CallAdapterFactory.create())
+                .build();
+        WeatherService weatherService = retrofit.create(WeatherService.class);
+        FiveDayForecast fiveDayForecast = weatherService.getFiveDayForecast("New York").blockingFirst();
```

Comment:
`blockingFirst()` should only be used in tests.
You need to use `subscribe()` like we did with Earthquakes.